shared/chat/inbox/row/small-team/bottom-line.js

```diff
@@ -93,14 +93,27 @@ class BottomLine extends PureComponent<Props> {
         style = baseStyle
       }
 
+      let snippetDecoration
+
+      // `snippetDecoration` will either be an explosion emoji, bomb emoji, or empty string.
+      // We want to use these emojis to render the correct custom icon.
+      switch (this.props.snippetDecoration) {
+        case '\u{1F4A5}': // Explosion (Collision) emoji (💥)
+          snippetDecoration = <Icon type="iconfont-boom" fontSize={16} />
+          break
+        case '\u{1F4A3}': // Bomb emoji (💣)
+          snippetDecoration = <Icon type="iconfont-bomb" fontSize={12} />
+          break
+      }
```

Comment:
we need a story for what happens if this changes
At a bare minimum if this.props.snippetDecoration doesn't match those 2 cases it should default: to passing the value straight through
aka
```
default:
  snippetDecoration = this.props.snippetDecoration
```

Reply:
added! also added some conditional rendering of the containing box, just in case we get a falsey value like empty string.